lib/css/atomic/_stacks-typography.less

```
@@ -151,6 +151,33 @@ h1, h2, h3, h4, h5, h6, p {
     overflow: hidden;
 }
 
+.v-truncate-fade {
+    overflow: hidden; 
+    -webkit-mask-image: linear-gradient(180deg, #000 @lh-md * 1em * 9, transparent); 
```

`9em` is `1em * 9` already, so you should just be able to go with that.

```suggestion
    -webkit-mask-image: linear-gradient(180deg, #000 @lh-md * 9em, transparent); 
```

---

In my head it was (one line height) * (9 lines), where "one line height" = `@lh-md * 1em`, so it didn't make sense to combine. I'll use a local variable so my mental model is a little more clear:

```
@one-lh: @lh-md * 1em;
-webkit-mask-image: linear-gradient(180deg, #000 @one-lh * 9, transparent); 
mask-image: linear-gradient(180deg,#000 @one-lh * 9,transparent); 
max-height: @one-lh * 12;
```